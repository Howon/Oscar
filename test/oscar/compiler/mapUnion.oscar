def main() => unit = {
  map<int, double> tMap1 = map<int, double>[0 -> 1.1, 4 -> 5.3, -3 -> 5.3, 12 -> 1.0];
  map<int, double> tMap2 = map<int, double>[0 -> 1.1, 4 -> 5.3, -7 -> 0.9];
  Println(Union(tMap1, tMap2) == map<int, double>[0 -> 1.1, 4 -> 5.3, -3 -> 5.3, 12 -> 1.0, -7 -> 0.9]);
}
