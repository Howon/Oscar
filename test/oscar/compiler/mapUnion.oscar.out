[0 -> 1.1, 4 -> 5.3, -3 -> 5.3, -7 -> 0.9]